src/libraries/System.Text.Json/gen/JsonSourceGenerator.Emitter.cs

```diff
@@ -18,21 +18,22 @@ namespace System.Text.Json.SourceGeneration
 {
     public sealed partial class JsonSourceGenerator
     {
+        private const string OptionsLocalVariableName = "options";
```

Comment:
Shouldn't this be grouped together with all the other constants?

Reply:
It's also used in the parser (it generates tiny snippet of code as it parses and stores that in the property)

Reply:
If the parser uses the same name in an independent context, then it might make sense to duplicate.